src/src/Helpers/PeriodicalActions.cpp

```diff
@@ -146,24 +146,28 @@ void runOncePerSecond()
     PluginCall(PLUGIN_CLOCK_IN, 0, dummy);
     if (Settings.UseRules)
     {
-      String event;
-      event.reserve(21);
-      event  = F("Clock#Time=");
-      event += node_time.weekday_str();
-      event += ',';
-
-      if (node_time.hour() < 10) {
-        event += '0';
-      }
-      event += node_time.hour();
-      event += ':';
+      // FIXME TD-er: What to do when the system time is not (yet) present?
+      if (node_time.systemTimePresent()) {
+        String event;
+        event.reserve(21);
+        event += F("Clock#Time=");
+        event += node_time.weekday_str();
+        event += ',';
+
+        if (node_time.hour() < 10) {
+          event += '0';
+        }
+        event += node_time.hour();
+        event += ':';
+
+        if (node_time.minute() < 10) {
+          event += '0';
+        }
+        event += node_time.minute();
```

Comment:
Might the use of `event += node_time.getTimeString(':', false);` possibly be a little more compact?